stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -316,18 +318,15 @@ internal class ChatDomainImpl internal constructor(
         logger.logI("Initializing ChatDomain with version " + getVersion())
 
         // if the user is already defined, just call setUser ourselves
-        val current = userOverwrite ?: client.getCurrentUser()
+        val current = client.getCurrentUser()
         if (current != null) {
             setUser(current)
         }
-        // past behaviour was to set the user on the chat domain
-        // the new syntax is to automatically pick up changes from the client
-        if (userOverwrite == null) {
```

Comment:
Why did you just delete this? It was exposed to our customers, they could still use it. If we don't wanna let them pass `userOverwrite` in ChatDomain directly we should use our deprecation policy.